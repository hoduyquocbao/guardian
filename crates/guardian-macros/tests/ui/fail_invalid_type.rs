use guardian_macros::frame;

#[frame]
pub struct Invalid {
    id: u32,
    invalid: MyCustomType, // This should cause a compilation error
    data: rest,
} 