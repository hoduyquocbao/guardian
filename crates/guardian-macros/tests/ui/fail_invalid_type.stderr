error: Unsupported field type
 --> $DIR/fail_invalid_type.rs:5:5
  |
5 |     invalid: MyCustomType, // This should cause a compilation error
  |     ^^^^^^^^ 